### DISCUSSION TITLE
支付宝购买lantern一年版，付费后未升级为专业版

### DISCUSSION THREAD

**Discussion by nousername223** [getlantern/lantern-forum-cn#909] [Q&A]
支付宝购买lantern一年版本，付费后未升级为专业版，没有任何的邮件发给我，请尽快处理，谢谢
![Screenshot_20240930_000945](https://github.com/user-attachments/assets/8e587377-3ac2-4863-bb5a-426fb3aa565a)

**Comment by MaxMKMax** [comment]
@nousername223 您好. 我帮您提交问题上去让他们尽快解决. 请您也提供我账号邮箱.

**Reply by nousername223** [reply]
1026824895@qq.com

**Reply by MaxMKMax** [reply]
@nousername223 谢谢. 提交好了.

**Reply by nousername223** [reply]
请问什么时候可以搞好？

**Reply by MaxMKMax** [reply]
@nousername223 
您好. 这是您的激活码:
QPY86-QMYMY-M86PH-V78CY-FDRDH

**Comment by nousername223** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
